api/src/test/java/org/apache/iceberg/types/TestSerializableTypes.java

```diff
@@ -46,6 +46,7 @@ public void testIdentityTypes() throws Exception {
           Types.StringType.get(),
           Types.UUIDType.get(),
           Types.BinaryType.get(),
+          Types.UnknownType.get()
```

Comment:
@aihuaxu, why not add variant as well?

Reply:
Variant test is already added below. Since for variant, we need to test with isEqualTo() instead of isSameAs(). 

> public void testVariant() throws Exception {